orchagent/portsorch.cpp

```diff
@@ -1220,7 +1220,7 @@ void PortsOrch::doPortTask(Consumer &consumer)
             }
 
             Port p;
-            if (!getPort(alias, p))
+            if (!getPort(alias, p) && alias != "PortConfigDone")
```

Comment:
The PORT_TABLE contains two non-port "flag" entries: `PortConfigDone` and `PortInitDone`. You will probably want to ignore both.

We really need to devise a way to remove them from the table. They don't belong there.

Reply:
Joe, I agree with you, but in this PR I want to make a fast fix of the wrong error message.
I don't want to change the swss architecture in this PR

Reply:
Understood. For this PR I suggest you also ignore `PortInitDone`.

Changing the architecture is a TODO.

Reply:
We don't see `PortInitDone` because the code exits from the method as soon as it sees alias 'PortInitDone'.

Reply:
https://github.com/Azure/sonic-swss/blob/master/orchagent/portsorch.cpp#L1095

Reply:
Agree with you both, we should find a new location in redisDB for all this metadata, or identify a different channel to transfer this state. PortInitDone and PortConfigDone are not the only cases, we also have "CONFIG_DB_INITIALIZED" and i'm sure i'm missing one or two examples more.